showcase-docs/getting-started.md

```diff
@@ -341,6 +341,10 @@ The easiest and fastest method for getting started: Backstage Showcase app, runn
 
 7. Navigate to <http://localhost:3000>
 
+#### Note: Extension to Kubernetes Backend Plugin
+
+The Kubernetes plugin has a default API factory that work right out the box with a few `oidcProviders` for authentication, but `oidc.oidc` is not included. An additional utility API has been added to the `backstage-plugin-kubernetes` plugin to allow the use of `oidcTokenProvider: 'oidc'` ([More on oidcTokenProvider](https://backstage.io/docs/features/kubernetes/configuration/#clustersoidctokenprovider-optional)).
+
```

Comment:
Ah, we just need to be careful providing links to upstream.  Docs cannot use them as reference.  Can you inline the config snippet as well?

Reply:
I've made the changes, lmk how it looks!